Dockerfile

```
@@ -13,14 +13,14 @@ RUN addgroup ${GROUP} && \
     adduser -u ${USER_ID} -G ${GROUP} -h /app -D ${USER} 
 RUN mkdir -p /app && \
```

You could do this and remove the `RUN` on line 16:
```suggestion
RUN mkdir -p /app/scripts && \
```